ibtk/include/ibtk/FEDataManager.h

```diff
@@ -1078,6 +1145,16 @@ class FEDataManager : public SAMRAI::tbox::Serializable
     void collectActivePatchNodes(std::vector<std::vector<libMesh::Node*> >& active_patch_nodes,
                                  const std::vector<std::vector<libMesh::Elem*> >& active_patch_elems);
 
+    /*!
+     * Store the association between subdomain ids and patch levels.
+     */
+    SubdomainToLevelTranslation d_level_lookup;
```

Comment:
FWIW, although this is not totally consistent, I think we usually do not expose member variables directly in the public interface.

Reply:
Oh, duh, this is not in the public interface.